### PR TITLE
Update update wizard i18n

### PR DESCRIPTION
**Pull request by rsalevsky**
Update the wizard i18n with the last changes from @kwwii.

### CODE REVIEW

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 1]
```diff
@@ -609,63 +609,62 @@ en:
...
-        blockui: 'Stopping OpenStack services...'
+        blockui: 'Stopping OpenStack Services...'
         header: 'Stop OpenStack Services'
         description: |
-          After continuing the Upgrade by pressing 'Stop OpenStack Services' all services on the
-          client nodes will be stopped. This is the last chance to access any OpenStack Service. If
-          not done yet, please shut down all running VM instances in your cloud and backup
-          everything you need to use OpenStack APIs.
+          Continuing the Upgrade process by clicking 'Stop OpenStack Services' below will stop all
+          services on the client nodes. This is the last chance to access any OpenStack Service. If
```
Comment (vuntz) [inline crowbar_framework/config/locales/crowbar/en.yml]: s/Service/service/ I think

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 2]
```diff
@@ -609,63 +609,62 @@ en:
...
+          you haven't doen so yet, please shut down all VM instances in your cloud and backup
```
Comment (vuntz) [inline crowbar_framework/config/locales/crowbar/en.yml]: s/doen/done/

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 3]
```diff
@@ -609,63 +609,62 @@ en:
...
+        description: 'Please wait until all nodes are upgraded. This may take a long time.'
+        total_nodes: 'Nodes in Total'
+        left_nodes: 'Nodes Left'
+        failed_nodes: 'Nodes Failed'
         finished: |
-          All nodes have been upgraded to SLES 12 SP1. This wizard is finished.
-          As a final step of the upgrade you need to re-apply all openstack
-          barclamps now in the proper order.
+          All nodes have been upgraded to SLES 12 SP1. This wizard is finished. In order to finish
+          the upgrade process please re-apply all openstack barclamps in the proper order.
```
Comment (vuntz) [inline crowbar_framework/config/locales/crowbar/en.yml]: s/openstack/OpenStack/

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 4]
```diff
@@ -609,63 +609,62 @@ en:
...
         continue: 'Stop OpenStack Services'
-        blockui: 'Stopping OpenStack services...'
+        blockui: 'Stopping OpenStack Services...'
         header: 'Stop OpenStack Services'
         description: |
-          After continuing the Upgrade by pressing 'Stop OpenStack Services' all services on the
-          client nodes will be stopped. This is the last chance to access any OpenStack Service. If
-          not done yet, please shut down all running VM instances in your cloud and backup
-          everything you need to use OpenStack APIs.
+          Continuing the Upgrade process by clicking 'Stop OpenStack Services' below will stop all
```
Comment (aplanas) [inline crowbar_framework/config/locales/crowbar/en.yml]: /Upgrade/upgrade ?

**Comment by vuntz** [comment]
+1

**Comment by aplanas** [comment]
+1

**Pull request merged**